routes/index.js

```diff
@@ -25,10 +23,30 @@ router.get('/', function(req, res, next) {
   };
 
   /**
-   * Load the entries file
+   * Load the entries file from the database
    */
   mongoose.connect(mongoConnection);
     Entry.find({}, renderEntries);
 });
 
+//Raring Route -- Khaled
+router.post('/rating', function(req, res, next) {
+  const id = req.body.id;
+  const helpful = req.body.helpful  
+  const renderEntries = function(error, entry) {
+    if (error) {
+      console.log(error);
+      res.sendStatus(400)
```

Comment:
A `400` status usually indicates an authentication error (ie - "You are not allowed to do this."). In your case, I think you want to render a `500` status (ie - "The server ran into an error.").